understanding/22/accessible-authentication-minimum.html

```diff
@@ -13,6 +13,16 @@ <h2>Status</h2>
         <p>This understanding document is part of the <a href="https://w3c.github.io/wcag/guidelines/22/"><strong>draft</strong> WCAG 2.2 content</a>. It may change or be removed before the final WCAG 2.2 is published.</p>
     </section> 
 
+    <section id="brief">
+        <h2>In brief</h2>
+        <dl>
+            <dt>Goal</dt><dd>Make logins possible with less mental effort.</dd>
+            <dt>What to do</dt><dd>Don’t make people solve, memorize, or transcribe something to log in.</dd>
+            <dt>Why it's important</dt><dd>Some people with cognitive disabilities cannot memorize a username and password, or type them.</dd>
```

Comment:
```suggestion
            <dt>Why it's important</dt><dd>Some people with cognitive disabilities cannot solve puzzles, memorize a username and password, or manually type them.</dd>
```

Reply:
"manually" in my suggestion is perhaps redundant, but could do with something that conveys the idea of transcribing. maybe "retyping"?

Reply:
I think we can be a bit more specific to address the main use case
```suggestion
            <dt>Why it's important</dt><dd>Some people with cognitive disabilities cannot solve puzzles, memorize a username and password, or transcribe one-time passcodes.</dd>
```

Reply:
hummm - is there something simpler than "transcribe"?